app/Notifications/CFPsAreOpen.php

```diff
@@ -2,28 +2,28 @@
 
 namespace App\Notifications;
 
-use App\Conference;
 use Illuminate\Bus\Queueable;
+use Illuminate\Support\Collection;
 use Illuminate\Notifications\Notification;
 use Illuminate\Notifications\Messages\MailMessage;
 
-class CFPIsOpen extends Notification
+class CFPsAreOpen extends Notification
 {
     use Queueable;
 
     /**
      * @var Conference
      */
-    public $conference;
+    public $conferences;
 
     /**
      * Create a new notification instance.
      *
      * @param Conference $conference
      */
-    public function __construct(Conference $conference)
+    public function __construct(Collection $conferences)
```

Comment:
What if we drop this type hint so you could also pass an array in? Not that we need to at the moment, but I don't see a need to type hint it and lock it down, since the Blade template is just foreaching anyway.

Reply:
That makes sense.  I went ahead and deleted the type hint.